src/types.jl

```diff
@@ -175,6 +195,18 @@ SHA2_384_CTX() = SHA2_384_CTX(copy(SHA2_384_initial_hash_value), 0, zeros(UInt8,
 Construct an empty SHA2_512 context.
 """
 SHA2_512_CTX() = SHA2_512_CTX(copy(SHA2_512_initial_hash_value), 0, zeros(UInt8, blocklen(SHA2_512_CTX)), false)
+"""
+    SHA2_512_224_CTX()
+
+Construct an empty SHA2_512/224 context.
+"""
```

Comment:
Would be nice if these docstrings contained a pointer/link to an authorative place defining the hash (possibly taken from the comments in some of the test files?)

Reply:
Done.